nipoppy_cli/pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
 ]
 dependencies = [
     "boutiques",
+    "packaging",
```

Comment:
Could be added to the test dependencies instead of the global ones.

Reply:
Good point!